ASSUMPTIONS.md

```diff
@@ -99,3 +102,87 @@ As more statistical data is collected about slashing in different validator envi
+
+To change this value, update the `daily_pow_issuance` [System Parameter](./model/system_parameters.py).
+
+### Ethereum Network Upgrade Stage Assumptions
+
+#### Simulation Start Date
+
+The simulation start date can be set using the `date_start` [System Parameter](./model/system_parameters.py). By default, for time-domain analyses, the start date is set to the current date.
+
+#### EIP-1559 Activation Date
+
+EIP-1559 transaction pricing will be included as part of the [London mainnet upgrade](https://blog.ethereum.org/2021/07/15/london-mainnet-announcement/) on 4 August 2021. After this date, the simulation will activate the necessary EIP-1559 mechanisms.
+
+To change this value, update the `date_eip1559` [System Parameter](./model/system_parameters.py).
+
+#### Proof-of-Stake Activation Date ("The Merge")
+
+The Ethereum researcher Justin Drake estimates a "best guess" of 1 December 2021 for the Proof-of-Stake activation date in his [ETH supply analysis](https://docs.google.com/spreadsheets/d/1ZN444__qkPWPjMJQ_t6FfqbhllkWNhHF-06ivRF73nQ). Launch dates are hard to predict, and for more rigorous analysis a range of values should be tested.
+
+To change this value, update the `date_pos` [System Parameter](./model/system_parameters.py).
+
+## Ethereum Transaction & EIP-1559 Assumptions
+
+### Average Gas Used
+
+With the introduction of [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559) the current gas limit is replaced by two values:
+* a “long-term average target” equal to the current gas limit
+* a “hard per-block cap” which is twice the current gas limit
+
+The long-term average gas target per block is set to 15m gas; by default we assume the gas used per block will on average be equal to the gas target. To change this value, update the `gas_target_process` [System Parameter](./model/system_parameters.py).
+
+### Average Fee Cap (`max_fee_per_gas`)
+
+The [EIP-1559 proposal](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1559.md) defines a variable `max_fee_per_gas` to set a fee cap per transaction. This fee cap is the maximum fee in Gwei per gas a user will pay per transaction. This fee is made up of the base fee and a priority fee, where the base fee is burned and the priority fee is paid to miners/validators.
+
+Pre EIP-1559 the fee cap is equivalent to the gas price in Gwei per gas. We use the historical mean gas price over the past 12 months to set the average fee cap, which in this model is equivalent to the sum of the values from the `base_fee_process` and `priority_fee_process`.
+
+We use a static value of 100 Gwei per gas (historical mean gas price) to set the default values for the `base_fee_process` and `priority_fee_process`. See [Miner Extractable Value as Percentage of Average Fee Cap](#miner-extractable-value-as-percentage-of-average-fee-cap) for further related assumptions.
+
+### Miner Extractable Value as Percentage of Average Fee Cap
+
+TODO: get Roger to review and validate
```

Comment:
@rogervs could I get your review on the assumptions added, and specifically the following section?